aspnetcore/performance/caching/distributed.md

```diff
@@ -25,6 +26,8 @@ Distributed cache configuration is implementation specific. This article describ
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/distributed/samples/) ([how to download](xref:index#how-to-download-a-sample))
 
+The sample app is configured to use a distributed SQL Server cache. To reconfigure the app to use a distributed Redis cache, change the preprocessor directive at the top of the *Startup.cs* file to use Redis (`#define Redis // SQLServer`). For more information, see <xref:index#preprocessor-directives-in-sample-code>.
```

Comment:
Consider adding a *readme.md* file with instructions. I don't think these instructions belong here. Put similar comments in the code.